Cargo.toml

```
@@ -13,7 +13,8 @@ A library to acquire a stack trace (backtrace) at runtime in a Rust program.
 """
 autoexamples = true
 autotests = true
-edition = "2018"
+edition = "2021"
+rust-version = "1.58.0"
```

MSRV should be 1.69, actually, but this is what's currently in CI, so.
```suggestion
rust-version = "1.65.0"
```